tests/services/test_remote_transfer_service.py

```
@@ -55,6 +55,52 @@ async def test_process_pending_transfers_empty(self, db_session, monkeypatch):
+    async def test_process_pending_transfers_limits_concurrency(
+        self, db_session, remote_transfer_job_factory, monkeypatch
+    ):
+        """The scheduler only starts the configured number of transfers at once."""
+        monkeypatch.setattr(db_session, "close", lambda: None)
+        monkeypatch.setattr(
+            "app.services.remote_transfer_service.SessionLocal", lambda: db_session
+        )
+        jobs = [remote_transfer_job_factory(status="PENDING") for _ in range(3)]
+        started = []
+
+        async def fake_run_transfer(job_id):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `fake_run_transfer`

Add return type annotation: `None`

(ANN202)